roles/openshift-cluster-status/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: check if oc client is installed
+  command: which oc
+  register: oc_installed
+  ignore_errors: yes
+
+- name: check if kubeconfig exists
+  stat:
+    path: "{{ ansible_env.HOME }}/.kube/config"
+  register: kubeconfig
+
+- name: print debug msg if oc client or kubeconfig doesn't exist
+  debug:
+    msg: Skipping the ocp collection, cannot find kubeconfig at "{{ ansible_env.HOME }}/.kube/config" or oc client is not installed, please check.
+  when: ( oc_installed.rc != 0 or not kubeconfig.stat.exists )
+
+- block:
+    - name: get events in all namespaces
+      command: oc get ev --sort-by='.lastTimestamp' --all-namespaces -o json
+      register: ocp_get_events_all_namespaces
+
+    - name: get openshift nodes
+      shell: oc get nodes -o wide -o json
```

Comment:
Can you replace `shell` with `command`?  See first note at [1].

[1] https://docs.ansible.com/ansible/latest/modules/shell_module.html#notes

Reply:
This also seems like repeated information collected in the `openshift-nodes` role [1], except that this is for all nodes, not just one.  We might want to remove this node collection in favor of that one to avoid duplicate data.

[1] https://github.com/redhat-performance/stockpile/blob/master/roles/openshift-nodes/tasks/main.yml#L77

Reply:
> Can you replace `shell` with `command`? See first note at [1].
> 
> [1] https://docs.ansible.com/ansible/latest/modules/shell_module.html#notes

You are right, fixing it.

> This also seems like repeated information collected in the `openshift-nodes` role [1], except that this is for all nodes, not just one. We might want to remove this node collection in favor of that one to avoid duplicate data.
> 
> [1] https://github.com/redhat-performance/stockpile/blob/master/roles/openshift-nodes/tasks/main.yml#L77

Right, I did not notice the openshift-node role.